### PR TITLE
Add scroll for other types of panes

### LINKED ISSUE #8: Work in search results
Hello!

Can this extension works on page with project search results?

image

Thanks.

### PR DESCRIPTION
**Pull request by AlexWayfer**
* Project Search Results
* Settings
* Files Tree

Resolves #8.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -84,16 +109,25 @@ export default {
 			}
 			this.stopScroll();
 		} else {
-			let editor;
-			if (e.button === 1 && (editor = e.target.closest("atom-text-editor:not([mini])")) && this.editor !== editor) {
+			const pane = e.target.closest(
+				"atom-text-editor:not([mini]), " +
+				".results-view-container, " +
+				".settings-view .panels-item, " +
+				".tree-view"
```
Comment (UziTech) [inline index.js:116]: Could you add comments for what each of these is for?

Comment (UziTech) [inline index.js:116]: Could we make this easier to add to?

Maybe something like:
```js
const selectors = [
	"atom-text-editor:not([mini])", // editors
	".results-view-container", // project find results
	".settings-view .panels-item", // settings panel
	".settings-view .config-menu", // settings menu
	".tree-view", // tree view
];
const pane = e.target.closest(selectors.join(", "));
```

also add `".settings-view .config-menu"`

**Review by UziTech — DISMISSED** [review]
Looks great just one change.

**Comment by UziTech** [comment]
Thanks 🎉 This is great work. I am going to change the selectors to be a setting so people can add whatever they want instead of hard coding all of the selectors.

**Comment by AlexWayfer** [comment]
OK, thank you! Sorry, I'm not too experienced with Atom Plugins development.

**Comment by UziTech** [comment]
You did great 💯. Thanks you

**Pull request merged**